__tests__/app.test.js

```diff
@@ -101,4 +100,58 @@ describe("/api/articles", () => {
           });
         });
     });
-  });
\ No newline at end of file
+    test("GET:200 articles are sorted by date descending", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles).toBeSortedBy("created_at", { descending: true });
+        });
```

Comment:
The test above states that you are checking that the date is sortedby descending. but also asserting it here?

Are you testing this twice? If so do you need to?